ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -149,6 +149,21 @@ boundary server -config="${this.configFilePath || '<path>'}/pki-worker.hcl"`;
       .join('\n    ');
   }
 
+  getClusterIdFromURL() {
```

Comment:
```suggestion
  get clusterID() {
```

Reply:
I can't make this directly just `clusterId` as I still need the tracked value for the input field on the form. I still wanted the user to be able to modify it if they wanted to, but just auto-populate it.

Reply:
Oooh.  Got it.  The cluster ID from URL is just an initial value.